docker/env/hydra.sh

```diff
@@ -245,8 +244,6 @@ function run_in_docker () {
     $([[ -n "$HYDRA_DRY_RUN" ]] && echo echo) \
     $tool ${REMOTE_DOCKER_HOST} run --rm ${TTY_STDIN} --privileged \
         -h ${HOST_NAME} \
-        -l "TestId=${SCT_TEST_ID}" \
```

Comment:
Why removing SCT_TEST_ID?

Reply:
same as below, it's meant for when we were running lots of test at the same time on the builders

we aren't doing that anymore, hence those labels are not really needed, or helpful.

Reply:
Unittest fails cause both of them, and would need to amended, either way